src/plugins/console/public/application/containers/editor/monaco/utils/autocomplete_utils.ts

```diff
@@ -338,7 +338,16 @@ const getInsertText = (
   }
   let insertText = '';
   if (typeof name === 'string') {
-    insertText = bodyContent.endsWith('"') ? '' : '"';
+    const bodyContentLines = bodyContent.split('\n');
+    const currentContentLine = bodyContentLines[bodyContentLines.length - 1];
```

Comment:
nit: there is a variable `lineContentAfterPosition` in the function that calls `getInsertText` so that could potentially be used here instead of splitting the lines

Reply:
Thanks for the review and the suggestion! I checked the value in this `lineContentAfterPosition` variable but it seems to contain the text **after** the initial position of the cursor (before we start deleting characters) - in the PR example, this is `": {}`. However, in this function we need the text **up until** this position to determine what string needs to be inserted there. So I think we can't reuse this variable but please let me know if I missed something and you see some way to reuse it

Reply:
You're right, @ElenaStoeva! sorry, I didn't notice the variable was for the content after.